api/comments/serializers.py

```diff
@@ -40,6 +41,9 @@ class CommentSerializer(JSONAPISerializer):
     date_modified = ser.DateTimeField(read_only=True)
     modified = ser.BooleanField(read_only=True, default=False)
     deleted = ser.BooleanField(read_only=True, source='is_deleted', default=False)
+    is_abuse = ser.SerializerMethodField()
```

Comment:
Minor: It would be good to add `help_text` for these fields, but that doesn't need to block this PR.